test/commands/info/releasenotes/display.test.ts

```diff
@@ -97,12 +97,14 @@ describe('info:releasenotes:display', () => {
     expect(getInfoConfigStub.args[0][0]).to.equal('/root/path');
   });
 
-  it('logs warning if info config lookup fails', async () => {
+  it('throws an error if info config lookup fails', async () => {
     getInfoConfigStub.throws(new Error('info config error'));
 
-    await runDisplayCmd([]);
-
-    expect(uxWarnStub.args[0][0]).to.contain('info config error');
+    try {
+      await runDisplayCmd([]);
```

Comment:
just in case somebody modifies the stubs, it's good to have a throw after the await so it doesn't fail silently.  

Happens enough we have this:
https://github.com/forcedotcom/sfdx-core/blob/107070d43cfc223c7b4fb9bfa6c1ec1c10166b97/src/testSetup.ts#L527

Reply:
Nice, that's handy